### ISSUE TITLE
Make module actions in the modal tab accessible

### ISSUE THREAD

**Issue by rase-** [Automattic/jetpack#1013]
When a modal is opened with information about a module, tab navigation still happens under it. Instead module actions should be accessible with tab when the modal is active.

**Comment by zinigor** [comment]
I don't think this issue is going to be valid for much longer, feel free to reopen if you don't agree.
